src/fields/field.hh

```diff
@@ -98,6 +98,11 @@ public:
     	 * @p FieldBase<...>::function_factory.
     	 */
     	virtual FieldBasePtr create_field(Input::Record rec, const FieldCommon &field);
+
+    	/**
+    	 * Check if Input::Record accessor contains data of field given by input_name
```

Comment:
Add comment: Returns true when ever the method create_field returns non-null pointer, otherwise returns false. 
The difference is that the field is actually not created.